apps/desktop/desktop_native/src/powermonitor/unimplemented.rs

```diff
@@ -0,0 +1,9 @@
+use napi::threadsafe_function::{ErrorStrategy::CalleeHandled, ThreadsafeFunction};
+
+pub async fn on_lock(_: ThreadsafeFunction<(), CalleeHandled>) -> Result<(), Box<dyn std::error::Error>> {
+    return Ok(());
```

Comment:
```suggestion
    unimplemented!();
```

I think we should use `unimplemented!` here to panic in case this gets accidentally called on non-linux platforms. That way it's easier to see why it's  not working.

Reply:
Done!